### PR TITLE
(CONT-406) Fix for RHEL 7 compatibility

### PR DESCRIPTION
**Pull request by david22swan**
A previous PR had updated the security_crs.comf.erb file to a more modern version, however this has rendered it incompatible with RHEL 7. This PR introduces and if to solve the issue.

Also includes a quick fix to remove legacy facts from a template file.

### CODE REVIEW

File: templates/mod/security.conf.erb [hunk 1]
```diff
@@ -54,13 +54,13 @@
     <%- end -%>
     SecArgumentSeparator &
     SecCookieFormat 0
-<%- if scope.lookupvar('::osfamily') == 'Debian' -%>
+<%- if $facts['os']['family'] == 'Debian' -%>
```
Comment (ekohl) [inline templates/mod/security.conf.erb:57]: This is ERB, not EPP

**Review by ekohl — CHANGES_REQUESTED** [review]
I wouldn't mind converting to EPP eventually, but we're not there today.

File: templates/mod/security.conf.erb (hunk 1 above)
Comment (david22swan) [reply templates/mod/security.conf.erb:57]: crap, read the wrong guide

**Review by GSPatton — APPROVED** [review]
looks good to me

**Pull request merged**